formats/polus-tabular-to-feather-plugin/Dockerfile

```diff
@@ -0,0 +1,30 @@
+
+FROM labshare/polus-bfio-util:2.1.9
+
+# environment variables defined in labshare/polus-bfio-util
+# ENV EXEC_DIR="/opt/executables"
+# ENV DATA_DIR="/data"
+# ENV POLUS_EXT=".ome.tif"
+# ENV POLUS_LOG="INFO" # Change to WARNING for fewer logs, and DEBUG for debugging
+
+# Work directory defined in the base container
+# WORKDIR ${EXEC_DIR}
+
+#Create folders
+RUN mkdir -p ${EXEC_DIR} \
+    && mkdir -p ${DATA_DIR}/inputs \
+    && mkdir -p ${DATA_DIR}/outputs
+
+COPY VERSION ${EXEC_DIR}
+COPY src ${EXEC_DIR}/
+
+RUN pip3 install -r ${EXEC_DIR}/requirements.txt --no-cache-dir
+
+WORKDIR ${EXEC_DIR}
+
+RUN ls -la ${EXEC_DIR}/*
+
+RUN ls -la ${DATA_DIR}/*
```

Comment:
This seems unnecessary. The work directory is already set in the parent container.